### ISSUE TITLE
Image Annotations - Dental Clinic Module with Dental Chart

### ISSUE THREAD

**Issue by niyazrazak** [earthians/marley#506]
### Is your feature request related to a problem? Please describe.

I would like to request the addition of a Dental Clinic module in the Marley app, which would include a Dental Chart feature. This new module would allow dentists to visualize and manage patient dental records more effectively.

Example:
![Image2](https://github.com/user-attachments/assets/1e4aa3a2-05f3-4546-acb0-3e3a0c822f5d)

**Comment by kalungia** [comment]
@niyazrazak this is from Odoo for Almighty systems but it will be better to do a better design

**Comment by niyazrazak** [comment]
> [@niyazrazak](https://github.com/niyazrazak) this is from Odoo for Almighty systems but it will be better to do a better design

Yes, I’m aware it’s from Odoo; I shared it specifically as a reference for a feature request to develop a dental clinic module in the Marley app. If you believe you have the skills and can contribute with a better design, I encourage you to support the community by sharing your expertise. Constructive contributions are always welcome!
